weatherman.py

```diff
@@ -0,0 +1,223 @@
+from zipfile import ZipFile
+import os
+import getopt
+import sys
+import gc
+
+
+months = [None, 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+          'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+
+
+def integer_conversion(s):
```

Comment:
```suggestion
def convert_to_int(s):
```
We should use assertive language for functions that performs a specific actions